From cdea3e16c8b3b74b5f82891cf94a3b0d2c3b93ca Mon Sep 17 00:00:00 2001
From: Eric Sun <jian.sun@freescale.com>
Date: Wed, 11 Jul 2012 11:21:17 +0800
Subject: [PATCH 1736/2508] ENGR00216282 [MX6 SABREAUTO(ARD)] No mxs-perfmon.0
 directory

The problem is caused because "mx6_board_init" don't add the
corresponding device node. Problem resolved after add them.

Signed-off-by: Eric Sun <jian.sun@freescale.com>
---
 arch/arm/mach-mx6/board-mx6q_sabreauto.c |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/arch/arm/mach-mx6/board-mx6q_sabreauto.c b/arch/arm/mach-mx6/board-mx6q_sabreauto.c
index f49e99a..2bb915a 100644
--- a/arch/arm/mach-mx6/board-mx6q_sabreauto.c
+++ b/arch/arm/mach-mx6/board-mx6q_sabreauto.c
@@ -1608,6 +1608,10 @@ static void __init mx6_board_init(void)
 
 	/* Add PCIe RC interface support */
 	imx6q_add_pcie(&mx6_sabreauto_pcie_data);
+
+	imx6q_add_perfmon(0);
+	imx6q_add_perfmon(1);
+	imx6q_add_perfmon(2);
 }
 
 extern void __iomem *twd_base;
-- 
1.7.7.4

